### PR TITLE
Changes to build the operator manually and travisCI to pass in operator piece.

### PR DESCRIPTION
**Pull request by whowutwut**
```
[root@c943f4n01-pvt operator]# operator-sdk build csi-scale-operator
INFO[0005] Building OCI image csi-scale-operator
Sending build context to Docker daemon 43.92 MB
Step 1/8 : FROM quay.io/operator-framework/ansible-operator:v0.11.0
 ---> 9a7b807df8cf
Step 2/8 : MAINTAINER jdunham@us.ibm.com
 ---> Using cache
 ---> 5fc5eba820c8
Step 3/8 : LABEL name "IBM Spectrum Scale CSI Operator" vendor "ibm" version "1.0.1" release "1" run 'docker run ibm-spectrum-scale-csi-operator' summary "An Ansible based operator to run and manage the deployment of the IBM Spectrum Scale CSI Driver." description "An Ansible based operator to run and manage the deployment of the IBM Spectrum Scale CSI Driver."
 ---> Using cache
 ---> a540a1661b4c
Step 4/8 : COPY build/health_check.sh .
 ---> Using cache
 ---> 97e130a612f9
Step 5/8 : COPY licenses /licenses
 ---> Using cache
 ---> 8eb4115c8ab8
Step 6/8 : COPY watches.yaml ${HOME}/watches.yaml
 ---> Using cache
 ---> 1869254ccd10
Step 7/8 : COPY build/_output/bin/operator ${OPERATOR}
 ---> Using cache
 ---> e0a798c57a8b
Step 8/8 : COPY roles/ ${HOME}/roles/
 ---> Using cache
 ---> 51d785e50a5a
Successfully built 51d785e50a5a
INFO[0006] Operator build complete.
```

### CODE REVIEW

**Comment by whowutwut** [comment]
At this point the operator linting steps is succeeding... 
image

Let's see what happens with the build step

**Comment by whowutwut** [comment]
Build Succeeded 
image

**Comment by whowutwut** [comment]
Cool, there are some warnings in the molecule testing, but  right now we're getting green 

image


I'm not going  to do anymore into this PR, asking @mew2057 to review.

File: .travis.yml [hunk 1]
```diff
@@ -156,83 +154,6 @@ jobs:
...
-
-language: go
-go:
-  - "1.13"
-
-env:
-  global:
-    - REPO_OWNER=$(dirname $TRAVIS_REPO_SLUG)
-    - REPO_NAME=$(basename $TRAVIS_REPO_SLUG)
-
```
Comment (whowutwut) [inline .travis.yml]: @dunnevan Just  FYI,  I removed the driver piece, you probably should put it back in when you get to testing it, sorry!

**Comment by mew2057** [comment]
I did this in  a local branch, but I think we should collapse scorecard and molecule to speed up  the ci run.

**Pull request merged**